docs/components/best-practices/architecture/deciding-about-your-stack.md

```diff
@@ -9,83 +9,77 @@ tags:
+This best practice specifically targets Camunda 8. For Camunda 7, please refer to [Deciding about your Camunda 7 stack](../deciding-about-your-stack-c7/).
 :::
 
 ## The greenfield stack
 
-We like to give one greenfield stack recommendation, which is the stack you can simply use if there is no reason against it. And while we went through long and detailed discussions to come to this recommendation, it _doesn't_ mean that it is necessarily superior to alternative stacks. You can still feel confident if you go down another route (note below for alternative options).
-
-The stack looks pretty similar in various programming language. Please use the programming language your team is most familiar with. If in doubt, use Java or JavaScript.
+Our greenfield stack recommendation is a result of extensive discussions and evaluations. While not the only option, it is a solid choice if there are no specific reasons to choose an alternative. Your choice of programming language should align with your team's expertise; we suggest Java or JavaScript for their broad applicability and support.
 
 ### The Java greenfield stack
 
 ![greenfield stack architecture diagram](deciding-about-your-stack-assets/greenfield-architecture.png)
 
-Use the following stack:
+For a Java-based setup using Camunda 8 SaaS and Spring Boot, use the following stack:
 
-1. Use [Camunda 8 SaaS](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral) and create a cluster there
+1. **Camunda 8 SaaS:**
 
-1. Build your process solution project as a [Spring Boot](https://spring.io/projects/spring-boot) application, using the [Spring Zeebe](/docs/apis-tools/community-clients/spring.md).
+   1. **Sign up:** Users need to sign up for Camunda 8 SaaS. This can be done through the [Camunda SaaS registration page.](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral)
+   2. **Create a cluster:** After signing up, users should create a cluster. They can follow the guide on [Creating a Cluster in Camunda 8](/guides/create-cluster.md) which provides step-by-step instructions on setting up a new cluster in the Camunda 8 environment.
 
-1. Use [Maven](https://maven.apache.org/) as a build tool.
+2. **Spring Boot:**
 
-1. Use your favorite IDE, for example Visual Studio Code, IntelliJ or Eclipse.
+   1. **Develop a Spring Boot application:** Users should develop their process solutions as [Spring Boot](https://spring.io/projects/spring-boot) applications. This involves setting up a new Spring Boot project, either manually or using tools like [Spring Initializr](https://start.spring.io/).
+   2. **Integrate Spring Zeebe:** Integrate [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe) into the Spring Boot project by adding necessary dependencies to the project’s pom.xml file and configuring the application to use Camunda services.
```

Comment:
We have a lot of disclaimers around community clients since Spring Zeebe is not yet fully supported. We should add something here as well.

Reply:
Yeah, that's a good add; we can remove that when it becomes part of the Camunda official offering.